src/main/kotlin/org/jitsi/rtp/NewRawPacket.java

```diff
@@ -812,6 +792,7 @@ public void removeExtension()
     public void setBuffer(byte[] buffer)
     {
         super.setBuffer(buffer);
+        // TODO: Is this necessary here?
```

Comment:
Yes, because HeaderExtensions.headerExtension uses the buffer directly!

Reply:
yes, i noticed this later.  i actually decided to change the design slightly in the ported version (rather than storing its own buffer reference, HeaderExtensions always uses the one in RtpPacket).  i can explain more why i did it that way, but i think it works better in kotlin (as it avoids making this field nullable, plus i'm not sure there's a reason not to do it this way).